templates/_components/_image.twig

```diff
@@ -500,9 +501,9 @@
     'data-src': imageDefault.url,
 
   }:{}) | merge(imageTransforms is defined and imageTransforms|length > 1 ? {
-    sizes: imageSizesGuide ?? false,
-    srcset: not imageLazyload ? defaultSrcSet : false,
-    'data-sizes': imageLazyload and not (imageSizesGuide ?? true) ? 'auto',
+    sizes: not imageLazyload ? (imageSizesGuide ?? false),
+    srcset: not imageLazyload ? defaultSrcSet,
+    'data-sizes': imageLazyload ? 'auto',
```

Comment:
Should `data-sizes` be `false` if the image is not lazy loaded?

Reply:
Supplying a false value and not setting any value both are giving same result: no `data-sizes` attribute set on the tag. Should I set false for else case in `data-sizes`, `data-srcset` and `srcset`?

Reply:
Okay if the outcome is the same, then we don’t need to set `false`. But I hope you have verified this because my understanding so far was that only `false` will make the attribute not being set, and empty string would still print the attribute.

Reply:
I tested it on APU.

<img width="716" alt="Screenshot 2024-05-24 at 2 17 05 PM 1" src="https://github.com/miranj/craft-boilerplate/assets/30666052/4ef5ee1b-c86f-41ce-a88f-e21b3224088d">